sorc/chgres_cube.fd/input_data.F90

```diff
@@ -98,9 +103,13 @@ module input_data
  type(esmf_field), public        :: ustar_input_grid        ! fric velocity
  type(esmf_field), public        :: veg_type_input_grid     ! vegetation type
  type(esmf_field), public        :: z0_input_grid           ! roughness length
+ type(esmf_field), public        :: veg_greenness_input_grid ! vegetation fraction
+ type(esmf_field), public        :: lai_input_grid          ! leaf area index
+ type(esmf_field), public        :: max_veg_greenness_input_grid ! shdmax
+ type(esmf_field), public        :: min_veg_greenness_input_grid ! shdmin
 
- integer, public      :: lsoil_input=4  ! # of soil layers,
-                                                   ! # hardwire for now
+ integer, public      :: lsoil_input=4  ! # of soil layers, no longer hardwired to allow
```

Comment:
Is this variable used anymore?

Reply:
Yes, but it can now be changed by the code if it determines that the input data has 9 soil levels (e.g, HRRR data).